x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.tsx

```diff
@@ -164,6 +178,10 @@ export function AlertInstances({
 }
 export const AlertInstancesWithApi = withBulkAlertOperations(AlertInstances);
 
+function getPage(items: any[], pagination: Pagination) {
```

Comment:
Should this be typed, instead of `any`?  It's not clear from the usage whether or not the caller has the types (just looking at the PR)

Reply:
I was thinking of it as "I really don't care what's in a page, as long as it's an array of things.
This is the one legitimate usage I see for `any`.

Make sense? :)
We can use Generic to achieve the same thing but as I don't interact with the item in any way, it feels like a legitimate `any`.

Make sense?

Reply:
Ah, I see.  It appears react itself or perhaps EUI react components aren't ts-type friendly (yet?), so having types won't help.  I was **obviously** confused about seeing you use an `any` 😃😃😃😃